components/layout/display_list_builder.rs

```diff
@@ -1440,7 +1440,7 @@ impl FragmentDisplayListBuilding for Fragment {
     fn adjust_clipping_region_for_children(&self,
                                            current_clip: &mut ClippingRegion,
                                            stacking_relative_border_box: &Rect<Au>,
-                                           is_absolutely_positioned: bool) {
+                                           _is_absolutely_positioned: bool) {
```

Comment:
Sorry, I really mean cc @pcwalton who removed the uses of this argument in 1e884ddc6969989fd2837d5262ff8cceb0ca2f9f.